InputOutput.py

```diff
@@ -161,11 +161,16 @@ def rawParseArgs(program, parser = None, no_args = False) :
 
         
         if len(args) == 1:
-            with open(args[0]) as f:
-                args = []
-                for line in f:
-                    if line[0] != "-": line = "-" + line
-                    args.extend(re.split(r"[,|\s|\n]+",line))
+            if args[0] in ["-h", "-help", "--help"]:
+                if args[0] == "-help":
```

Comment:
@gregorgorjanc I have the code tested on my end and all these work. And I want to utilize the code of ``-h`` and ``--help`` that is already in the package ``argparse`` and that's the reason I added line 165.

Reply:
@XingerTang I see `argparse` already handles `-h` and `--help`! I think it would be best to be explicit and remove the 165 so that it's clear and hence we don't get confused in the future.

Reply:
@gregorgorjanc I am done with the modification and please take a look at it.